src/squeak/byte-take-iterator.js

```diff
@@ -1,13 +1,35 @@
+/**
+ * Let the passed iterator only take bytes up to a certain position.
```

Comment:
This is a weird way to start a comment describing a class. Maybe:

```suggestion
 * An iterator that only takes bytes up to a certain position.
```

Reply:
Alternatively, maybe just remove this line completely. The comment below makes sense without it.